### PR TITLE
Joker4mas patch 1

### PR DESCRIPTION
**Pull request by Joker4mas**
I fixed the CTA section of Services. Kindly approve and merge it. Thank you

### CODE REVIEW

File: Autoaid/src/main.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
 import { BrowserRouter } from "react-router-dom";
+
```
Comment (Kennymartin16) [inline Autoaid/src/main.tsx:4]: Not necessary

File: Autoaid/src/Components/CTA.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          className="cta-text bg-primary max-w-xl
+          z-50 p-12 h-96
+         relative left-96"
+        >
+          <h2 className="text-5xl font-extrabold py-6 ">
+            Get in touch with us for your service related query
+          </h2>
+
+          <button className="bg-dark-gray text-white hover:bg-steal-dark py-2 px-10 rounded-md my-6">
+            Book Now
```
Comment (Kennymartin16) [inline Autoaid/src/Components/CTA.tsx:24]: This should be Contact Us or was it intentional?

File: Autoaid/src/App.tsx [hunk 3]
```diff
@@ -1,24 +1,24 @@
-import React from 'react';
-import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
-import Blog from './pages/Blog';
-import Article from './pages/Article';
-import Header from "./Components/header";
-import Footer from "./Components/footer";
+import React from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Blog from "./pages/Blog";
+// import Article from "./pages/Article";
```
Comment (Kennymartin16) [inline Autoaid/src/App.tsx:4]: Why commenting out the Article import?

File: Autoaid/src/Components/CTA.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+function CTA() {
+  return (
+    <>
+      <div className="flex flex-col md:flex-row relative">
+        <div className="cta-car z-40 absolute left-0">
+          <img
+            src="/service-Images/cta-car.png"
```
Comment (Kennymartin16) [inline Autoaid/src/Components/CTA.tsx:9]: This is a public component and its Images should be seperate from the service image folder
I have finished it from my end but haven't push it.

**Review by Kennymartin16 — CHANGES_REQUESTED** [review]

**Review by Kennymaartin — APPROVED** [review]

**Review by Kennymaartin — APPROVED** [review]

**Review by Kennymaartin — APPROVED** [review]

**Review by Kennymaartin — APPROVED** [review]

**Pull request merged**